server/src/dtos/memory.dto.ts

```diff
@@ -0,0 +1,65 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsEnum, IsObject } from 'class-validator';
+import { AssetResponseDto, mapAsset } from 'src/dtos/asset-response.dto';
+import { MemoryEntity, MemoryType } from 'src/entities/memory.entity';
+import { ValidateBoolean, ValidateDate, ValidateUUID } from 'src/validation';
+
+type MemoryData = any;
+
+class MemoryBaseDto {
+  @ValidateBoolean({ optional: true })
+  isSaved?: boolean;
```

Comment:
Not really. Not passing it in an update call would change it from undefined to false and then it would always set it to false, even if it was previously set to true.

Reply:
Oh yeah, I was thinking about creation but updates would require it to be optional.

Reply:
I usually just set the default value in the database and then make everything optional. Especially when it defaults to false, undefined is false when it is being checked like that.